rd-net/RdFramework/Impl/RdSignal.cs

```diff
@@ -63,7 +63,7 @@ protected override void Init(Lifetime lifetime)
     public override void OnWireReceived(UnsafeReader reader)
     {
       var value = ReadValueDelegate(SerializationContext, reader);
-      ReceiveTrace?.Log($"{this} :: value = {value.PrintToString()}");
+      ReceiveTrace?.Log($"{this} :: value = {value?.PrintToString()}");
```

Comment:
_(Same as above)_ `PrintToString` is supposed to handle null better (by printing `"<null>"`), so could we leave this as is?